transport_nantes/transport_nantes/settings.py

```diff
@@ -272,3 +275,10 @@
 CELERY_TASK_TRACK_STARTED = True
 CELERY_TASK_TIME_LIMIT = 30 * 60
 CELERY_BROKER_URL = "amqp://localhost"
+
+ROLLBAR = {
+    'access_token': settings_local.ROLLBAR_ACCESS_TOKEN,
+    'environment': 'dev' if DEBUG else 'production',
+    'root': os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
```

Comment:
Why not `settings_local.BASE_DIR`?

Reply:
Because settings_local.py states :

> # NOT AVAILABLE IN PRODUCTION
> # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
> BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))

base dir is absent in infra afaik